tools/netlist/tests/NameTests.cpp

```diff
@@ -55,3 +55,40 @@ endmodule
     auto netlist = createNetlist(compilation);
     CHECK(netlist.numNodes() > 0);
 }
+
+TEST_CASE("Ignore concurrent assertions") {
+    // Test that unused modules are not visited by the netlist builder.
```

Comment:
Comment is copy pasta from the previous test case.

Reply:
fixed locally, will be pushed when I have more meaningful changes later.